wolfcrypt/src/asn.c

```diff
@@ -26849,12 +26869,15 @@ static int DecodeAsymKeyPublic(const byte* input, word32* inOutIdx, word32 inSz,
     *pubKeyLen = inSz - *inOutIdx;
     XMEMCPY(pubKey, input + *inOutIdx, *pubKeyLen);
 #else
+    len = inSz - *inOutIdx;
```

Comment:
What if inOutSz is larger than inSz? I don't see it checked in the ASN template code.